Agregué el botón para crear una nueva cuenta

RodrigoVite opened the pull request:
Se usa la misma pantalla del login, solo llenando los campos y presionando el boton debería crearse la nueva cuenta. 
Chécalo y me dices que te parece, por fa.
Además agregué los cambios planteados en el pull request anterior

RodrigoVite committed:
Agregué el boton para crear una nueva cuenta

RodrigoVite committed:
Update login.dart

DavidEstrada99 commented:
El programa funciona bien, pero me gustaría que se implementara la verificación de correo. si pudiera pedirse una verificación antes de entrar a la aplicación general, funcionaria muy bien

RodrigoVite committed:
Pantalla de espera de verificación de email

Odié esta pantalla, por un try-catch me atoré una hora, tal vez más jajajajajajajaaj

RodrigoVite commented:
Checa las ultimas modificaciones que hice

DavidEstrada99 reviewed (approved):
Todo funciona correctamente
